docker/nginx/nginx.conf

```diff
@@ -0,0 +1,125 @@
+user                    www-data www-data;
+worker_processes        5;
+error_log               /etc/nginx/logs/error.log;
+pid                     /etc/nginx/nginx.pid;
+worker_rlimit_nofile    8192;
+
+events {
+  worker_connections    4096;
+}
+
+http {
+    include    /etc/nginx/mime.types;
+    include    /etc/nginx/proxy.conf;
+    include    /etc/nginx/fastcgi.conf;
+
+    default_type application/octet-stream;
+    server_names_hash_bucket_size 128;
+    log_format scripts '$document_root$fastcgi_script_name > $request';
+
+    sendfile            on;
+    sendfile_max_chunk  1m;
+    tcp_nopush          on;
+    tcp_nodelay         on;
+    keepalive_timeout   65;
+
+    gzip                on;
+    gzip_min_length     1000;
+    gzip_buffers        4 8k;
+    gzip_types          text/plain application/x-javascript text/css image/png image/jpeg image/gif image/x-icon image/svg+xml;
+    gzip_vary           on;
+
+    map $host $x_css_protection {
+        default "1; mode=block";
+    }
+    map $host $x_content_type_options {
+        default "nosniff";
+    }
+    map $host $content_security_policy {
+        default "default-src 'self' https://www.googleapis.com; script-src 'self' 'unsafe-inline'; img-src 'self' images.gewis.nl data:; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/; font-src 'self' https://fonts.gstatic.com/; frame-src 'self'; object-src 'none'; frame-ancestors 'self'; form-action 'self'";
+    }
```

Comment:
`https://www.googleapis.com` should not be in the `default-src` directive. It defeats the whole purpose of having a CSP in the first place if you add a website you don't manage to all non-existing directives.

To make it work, it should go under `connect-src`. However, you also need `gewis.nl` to load the templates (making it `conntect-src 'self' https://www.googleapis.com`).

**An even bigger problem is that it still won't work, because it requires `'unsafe-eval'` in the `script-src` directive.**

Reply:
I fixed the first.

Regarding the latter: we would need to change some things on the website to make it work then. Since we do not have a CSP at the moment, it's still an improvement.

Reply:
What happens without `unsafe-eval` in `script-src`:
![image](https://user-images.githubusercontent.com/4983571/120781296-3d6fcc00-c529-11eb-88be-3192d6b435da.png)

Reply:
Yeah, I noticed. It works now!